com.unity.multiplayer.mlapi/Runtime/SceneManagement/SceneEventData.cs

```diff
@@ -0,0 +1,496 @@
+using System.Collections.Generic;
+using System;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+namespace Unity.Multiplayer.Netcode
+{
+    [Serializable]
+    public class SceneEventData : IDisposable
+    {
+        public enum SceneEventTypes
+        {
+            Event_Load,                 //Server to client load additive scene
```

Comment:
Is it maybe worth including S2C/C2S in the name of these to make this clearer?

Reply:
Adjusted the enum to reflect the above suggestion.